npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker_spec.rb

```diff
@@ -1426,14 +1443,15 @@
         let(:registry_listing_url) { "https://registry.npmjs.org/transitive-dependency-locked-by-intermediate" }
 
         it "correctly updates the transitive dependency" do
-          expect(checker.send(:updated_dependencies_after_full_unlock)).to eq([
+          expect(checker.send(:updated_dependencies_after_full_unlock)).to eq_with_metadata([
```

Comment:
Nice, I would maybe have chosen `eq_including_metadata` but I'm not great at English 😛.

Reply:
I like that wording better as well so updated to use it!